guides/common/modules/proc_remove-hosts-from-red-hat-insights-inventory-and-satellite-simultaneously.adoc

```diff
@@ -0,0 +1,15 @@
+[id="remove-hosts-from-red-hat-insights-inventory-and-satellite-simultaneously_{context}"]
+= Remove Hosts from Red Hat Insights Inventory and Satellite simultaneously
```

Comment:
```suggestion
= Configuring Automatic Removal of Hosts from the Insights Inventory
```
I'm thinking about a simpler, clearer, and more precise title here. How about this?

Reply:
Yes, better. Thanks :+1: